website/source/docs/agent/encryption.html.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: "docs"
+page_title: "Encryption"
+sidebar_current: "docs-agent-encryption"
+description: |-
+  Learn about the encryption in Nomad.
+---
+
+The Nomad agent supports encrypting all of its network traffic. There are
+two separate encryption systems, one for gossip traffic and one for RPC.
+
+# Gossip Encryption
```

Comment:
It's not proper to have two h1 tags on the same page. There should only be one h1 tag (encryption) and then multiple h2 tags for each type of encryption (gossip, rpc, etc)